src/Raven.Server/ServerWide/Maintenance/ClusterMaintenanceSupervisor.cs

```diff
@@ -248,7 +249,16 @@ private void ListenToMaintenanceWorker()
 
                                 var report = BuildReport(rawReport);
                                 timeoutEvent.Defer(_parent._leaderClusterTag);
-
+                                foreach (var name in report.Report.Keys.ToList())
+                                {
+                                    var dbReport = report.Report[name];
+                                    if (dbReport.Status == DatabaseStatus.NoChange)
+                                    {
+                                        report.Report[name] = _lastSuccessfulReceivedReport.Report[name];
```

Comment:
Is this correct? It looks like we'll miss out on the things that are updated, no? Like the replication stats?

Reply:
Fixed.